README.md

```diff
@@ -0,0 +1,11 @@
+## Hello, World
+<hr>
+A simple program, and an example of how to structure a python project.
+
+## Usage
+<br>
```

Comment:
Shouldn't need explicit `<br>` in markdown. You can force a line break either by adding a blank line between in the markdown or by adding two spaces at the end of the line where you want a line break.

Reply:
Ok